### DISCUSSION TITLE
Why is my most used languages not showing up?

### DISCUSSION THREAD

**Discussion by ENTPRESTIGIOUS** [anuraghazra/github-readme-stats#1892] [Q&A] [answered]
**Describe the bug**

**My most used languages aren't showing up with this link. It used to work in the past: 
https://github-readme-stats.vercel.app/api/top-langs/?username=entprestigious**

**Expected behaviour**

**It should display something.**

**Comment by rickstaa** [comment, marked as answer]
@ENTPRESTIGIOUS Thanks for your question. I checked your account and it seems that the card is displaying the correct result we receive from the GraphQL API. You can check this out yourself using the following syntax in the graphql Explorer:

```graphql
query userInfo($login: String!) {
  user(login: $login) {
    # fetch only owner repos & not forks
    repositories(ownerAffiliations: OWNER, isFork: false, first: 100) {
      nodes {
        name
        languages(first: 10, orderBy: {field: SIZE, direction: DESC}) {
          edges {
            size
            node {
              color
              name
            }
          }
        }
      }
    }
  }
}
```

```graphql
{"login": "ENTPRESTIGIOUS"}
```

Although you have code repositories, those repositories don't have language information linked to them. It looks like all your repositories only contain binary files or markdown. Those are not recognized by [linguists](https://github.com/github/linguist) as languages (see https://github.com/github/linguist/issues/5951).

**Comment by lbharath-07** [comment]
i have created a repository in my github but my github profile is not showing languages i used an you please solve this issue for me
my repo : https://github.com/lbharath-07/chatApp

**Comment by lbharath-07** [comment]
i have created a repository in my github but my github profile is not showing languages i used an you please solve this issue for me
my repo : https://github.com/lbharath-07/chatApp
can you please help me dude  @lildude

**Reply by lildude** [reply]
@lbharath-07 You should really start a new discussion as this one has been solved for over a year. Push another change. 

The language analysis only occurs on the default branch when a push is made. Your old default branch (`main`) has no code and that's where the last analysis took place. You have since changed default branches (now `master`) but not pushed any changes to that branch since switching. Pushing another change should kick things off. If not, open a new discussion at https://github.com/github-linguist/linguist/discussions
